client/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -1,251 +1,277 @@
+    this.OvhApiXdslTasksCurrent = OvhApiXdslTasksCurrent;
+    this.Toast = Toast;
+    this.ToastError = ToastError;
+    this.PACK = PACK;
+    this.PACK_IP = PACK_IP;
+    this.REDIRECT_URLS = REDIRECT_URLS;
+  }
+
+  $onInit() {
+    this.packName = this.$stateParams.packName;
+
+    this.$scope.loaders = {
+      details: true,
+      tasks: true,
+      deconsolidation: true,
+      xdsl: true,
+    };
+
+    this.$scope.access = {
+      xdsl: null,
+      tasks: { current: {} },
+    };
+
+    this.$scope.constants = {
+      rangeOfBaseIpv4IP: this.PACK_IP.baseIpv4Range,
+    };
+
+    this.init();
+  }
+
+  init() {
+    this.number = this.$stateParams.number;
+
+    this.getLinesDetails();
 
     /* eslint-disable max-len */
-    $templateCache.put('pack-xdsl-access-tooltip-mac.html', '<div class="tooltip-description" data-translate="xdsl_details_mac_address_description"></div>');
-    $templateCache.put('pack-xdsl-access-tooltip-dslam.html', '<div class="tooltip-description" data-translate="xdsl_access_dslam_reset_description"></div><div class="text-warning" data-translate="xdsl_access_dslam_reset_warning"></div>');
-    $templateCache.put('pack-xdsl-access-tooltip-lnsApply.html', '<div class="tooltip-description" data-translate="xdsl_access_lns_ratelimit_description"></div><div class="text-warning" data-translate="xdsl_access_lns_ratelimit_warning"></div>');
-    $templateCache.put('pack-xdsl-access-tooltip-lns.html', '<div class="tooltip-description" data-translate="xdsl_access_lns_description"></div><div class="text-warning" data-translate="xdsl_access_lns_warning"></div>');
-    $templateCache.put('pack-xdsl-access-tooltip-deconsolidation.html', "<div class=\"tooltip-description\" data-ng-bind=\" ('xdsl_access_deconsolidation_warning_' + XdslAccess.lineDetails.deconsolidation) | translate\"></div>");
-    $templateCache.put('pack-xdsl-access-tooltip-ipDelete.html', '<div class="tooltip-description" data-translate="xdsl_details_ips_remove_only_extra"></div>');
-    $templateCache.put('pack-xdsl-access-tooltip-ips.html', '<div class="tooltip-description" data-translate="xdsl_access_ipv6_description"></div><div class="text-warning" data-translate="xdsl_access_ipv6_warning"></div>');
-    $templateCache.put('pack-xdsl-access-tooltip-dslamProfile.html', '<div class="text-left"><p data-translate="xdsl_access_profile_tooltip_interleaved"></p><p data-translate="xdsl_access_profile_tooltip_fast"></p><p data-translate="xdsl_access_profile_tooltip_ginp"></p><p data-translate="xdsl_access_profile_tooltip_auto"></p><p data-translate="xdsl_access_profile_tooltip_snr"></p><p class="text-warning" data-translate="xdsl_access_profile_tooltip_time"></p></div>');
+    this.$templateCache.put('pack-xdsl-access-tooltip-dslam.html', '<div class="tooltip-description" data-translate="xdsl_access_dslam_reset_description"></div><div class="text-warning" data-translate="xdsl_access_dslam_reset_warning"></div>');
```

Comment:
Only mackarel ! I understand why you didn't change it but this deserves a ticket so someone actually changes it once and for all